Makefile

```diff
@@ -241,6 +253,12 @@ bundle-hub: manifests kustomize operator-sdk ## Generate hub bundle manifests an
 		--patch '[{"op": "add", "path": "/metadata/annotations/olm.skipRange", "value": "$(SKIP_RANGE)"}]' && \
 		$(KUSTOMIZE) edit add patch --name ramen-hub-operator.v0.0.0 --kind ClusterServiceVersion\
 		--patch '[{"op": "replace", "path": "/spec/replaces", "value": "$(REPLACES)"}]'
+	$(SED_CMD) -e "s,channelName: alpha,channelName: $(DEFAULT_CHANNEL)-test," -i config/hub/manifests/$(IMAGE_NAME)/ramen_manager_config_append.yaml
```

Comment:
Yes, this is ugly, but all kustomize failed to get em across the line here.

I could edit the base `_append.yaml` file in a copy and cat it to the core config file, so that we do not have a repo file touched during  a make.